nightfall-optimist/src/app.mjs

```diff
@@ -12,8 +17,17 @@ import {
   debug,
 } from './routes/index.mjs';
 
+const environment = config.ENVIRONMENTS[process.env.ENVIRONMENT] || config.ENVIRONMENTS.localhost;
+
 const app = express();
 
+const ethPrivateKey = environment.PROPOSER_KEY;
+const { address } = web3.eth.accounts.privateKeyToAccount(ethPrivateKey);
+
+app.set('ethPrivateKey', ethPrivateKey);
```

Comment:
i guess im still confused why you read `environment` and do `app.set('ethAddress')` and `app.set('ethPrivateKey'`) here, and not just read `environment` in `index.mjs`